mmdet/core/mask/structures.py

```diff
@@ -1069,3 +1069,32 @@ def polygon_to_bitmap(polygons, height, width):
     rle = maskUtils.merge(rles)
     bitmap_mask = maskUtils.decode(rle).astype(np.bool)
     return bitmap_mask
+
+
+def bitmap_to_polygon(bitmap):
+    """Convert masks from the form of bitmaps to polygons.
+
+    Args:
+        bitmap (ndarray): masks in bitmap representation.
+
+    Return:
```

Comment:
list(ndarray) -> list[ndarray], and it's better to unify the following two lines like `param_name (type): ....` or `type: ....`

Reply:
It should be the form of `type:`